viewer/static/js/app/reducers/metric-metadata-reducer.js

```diff
@@ -14,7 +14,7 @@ const getHoverStrings = (metadata) => {
   for (const id in metadata) {
     if (metadata.hasOwnProperty(id)) {
       var tooltip = metadata[id].tooltip;
-      strings['id-' + id] = tooltip ? tooltip : '{pop}: x={x}, y={y}%'
+      strings['id-' + id] = tooltip ? tooltip : '{pop}: x={x}, y={y}%, n={n}'
```

Comment:
Is it possible to "prettify" the number a little so large numbers like `220012` look like `220,012`?